LICENSE

```diff
@@ -534,4 +534,68 @@ ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 ========================================================================
+This projects includes libarchive bundle (https://www.libarchive.org)
+which is available under a BSD License by Tim Kientzle and others
+
```

Comment:
archive_getdate.c which is still bundled with this commit is listed as public domain and we should have appropriate attribution in our license

Reply:
Fixed in d2e7e34ab8b331ac484b9b16bd51455799a1502b